### PR TITLE
add container-llama bundle

### PR DESCRIPTION
**Pull request by arnej27959**
initial skeleton
@lesters please review
@gjoranv @bjorncs FYI

### CODE REVIEW

File: container-llama/pom.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      <groupId>com.yahoo.vespa</groupId>
+      <artifactId>jdisc_core</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <!-- compile -->
+    <dependency>
+      <groupId>de.kherud</groupId>
+      <artifactId>llama</artifactId>
+      <version>2.3.5</version>
```
Comment (gjoranv) [inline container-llama/pom.xml:38]: Please add a version property to parent pom for automatic upgrades.

**Review by gjoranv — COMMENTED** [review]
Nice!

**Review by lesters — COMMENTED** [review]
LGTM! But a SD error:

```
14:53:55 CMake Error at container-llama/cmake_install.cmake:46 (file):
14:53:55   file INSTALL cannot find
14:53:55   "/sd/workspace/src/github.com/vespa-engine/vespa/container-llama/target/container-llama.jar":
14:53:55   No such file or directory.
14:53:55 Call Stack (most recent call first):
14:53:55   cmake_install.cmake:186 (include)
```

File: parent/pom.xml [hunk 2]
```diff
@@ -593,6 +593,12 @@
                 <artifactId>onnxruntime</artifactId>
                 <version>${onnxruntime.vespa.version}</version>
             </dependency>
+           <dependency>
+               <groupId>de.kherud</groupId>
+               <artifactId>llama</artifactId>
+               <version>${java-llama.vespa.version}</version>
+               <scope>compile</scope>
```
Comment (gjoranv) [inline parent/pom.xml:600]: Please remove scope from the dependencyManagement. It should contain version only.

File: container-llama/pom.xml [hunk 3]
```diff
@@ -35,7 +35,7 @@
     <dependency>
       <groupId>de.kherud</groupId>
       <artifactId>llama</artifactId>
-      <version>2.3.5</version>
+      <version>${java-llama.vespa.version}</version>
```
Comment (gjoranv) [inline container-llama/pom.xml:38]: Version is managed from parent, so please remove.

File: dependency-versions/pom.xml [hunk 4]
```diff
@@ -124,6 +124,7 @@
         <netty.vespa.version>4.1.107.Final</netty.vespa.version>
         <netty-tcnative.vespa.version>2.0.65.Final</netty-tcnative.vespa.version>
         <onnxruntime.vespa.version>1.17.1</onnxruntime.vespa.version>
+        <java-llama.vespa.version>2.3.5</java-llama.vespa.version>
```
Comment (gjoranv) [inline dependency-versions/pom.xml:127]: Assuming this should not be exposed to customers, please move the property to `parent/pom.xml`

**Review by gjoranv — CHANGES_REQUESTED** [review]

File: dependency-versions/pom.xml (hunk 4 above)
Comment (arnej27959) [reply dependency-versions/pom.xml]: it looks to me like it belongs in this file, under
_Dependencies used internally in Vespa, not visible for users_

**Review by gjoranv — DISMISSED** [review]

File: vespa-dependencies-enforcer/allowed-maven-dependencies.txt [hunk 5]
```diff
@@ -30,6 +30,8 @@ com.google.jimfs:jimfs:${jimfs.vespa.version}
 com.google.protobuf:protobuf-java:${protobuf.vespa.version}
 com.ibm.icu:icu4j:${icu4j.vespa.version}
 com.microsoft.onnxruntime:onnxruntime:${onnxruntime.vespa.version}
+de.kherud:llama:${kherud.llama.vespa.version}
+org.jetbrains:annotations:24.0.1
```
Comment (bjorncs) [inline vespa-dependencies-enforcer/allowed-maven-dependencies.txt:34]: Please exclude jetbrains annotations. It's purely a compile time dependency - the annotation are not required at runtime.

**Review by bjorncs — COMMENTED** [review]
👍

**Pull request merged**